Extensions/old_reblogs.js

```diff
@@ -0,0 +1,90 @@
+//* TITLE Restore Old Reblogs **//
+//* VERSION 1.2.1 **//
+//* DESCRIPTION	Restores the old reblogging style on the Dashboard **//
+//* DEVELOPER eli <eli@csh.rit.edu> (forked from Editable Reblogs by dlmarquis) **//
```

Comment:
XKit automatically makes a link in the control panel to github.com/[developer]. You might want to just input your GitHub username here.